src/main/resources/templates/pps/bankTransferPenaltyReference.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
```

Comment:
Please can you change your HTML code style settings via `~/.editorconfig` preferably and reformat the html files you've added for consistency with other files in penalty-payment-web?

```
[*.html]
indent_size = 4
tab_width = 4
ij_continuation_indent_size = 8
```

Reply:
Updated the indent part